metrics/storage/fio-k8s/scripts/dax-compare-test/compare-virtiofsd-dax.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
```

Comment:
Document that this test requires to run in a single node clusterm  `scheduler` and `worker` are the same node.